src/portal/lib/src/tag/tag-detail.component.ts

```diff
@@ -128,25 +109,60 @@ export class TagDetailComponent implements OnInit {
       : "TAG.ANONYMITY";
   }
 
-  public get highCount(): number {
-    return this._highCount;
+  get criticalCount(): number {
```

Comment:
Please add description comment for the public method.

Reply:
fixed